docs/1_installation.md

```diff
@@ -1,14 +1,26 @@
 # ⬇️ Installation
 
-In order to use ReVanced on your Android device, ReVanced Manager must be installed.
+To use ReVanced on your Android device, ReVanced Manager have to be install, 
+refer to the [Prerequisites](0_prerequisites.md) if haven't already.
 
-## ✅ Installation steps
+## 🪜 Installation steps
 
-1. Download the latest version of ReVanced Manager from [here](https://github.com/revanced/revanced-manager/releases/latest)
+1. Get the latest version of ReVanced Manager from the [ReVanced site][Official ReVanced Download].
 2. Install ReVanced Manager
 
+### ✒️ Verifying authenticity of ReVanced Manager
+
+> [!NOTE]
+> It's always advisable that you download from trusted sources like the [ReVanced site][Official ReVanced Download]
+> or GitHub releases as it's the safest way to download ReVanced without potential risk of malware.
+
+To verify if your ReVanced Manager is provided without sign of tampering, 
+we have signed the APK with hashes corresponding to `b6362c6ea7888efd15c0800f480786ad0f5b133b4f84e12d46afba5f9eac1223` *unless otherwise stated*.
+
```

Comment:
This does not seem like enough information for a user to successfully verify an APK.

Reply:
Yes, the official Android docs to verify the APK should be referenced and/or the official documented method should be mentioned here.